src/lingrex/trimming.py

```diff
@@ -0,0 +1,201 @@
+"""
+Trimming functionalities in lingrex.
+"""
+import random
+import collections
+from lingpy.sequence.sound_classes import token2class
+
+
+def revert(alms):
+    return [[row[i] for row in alms] for i in range(len(alms[0]))]        
+
+
+def get_skeleton(alms, gap="-"):
+    return [token2class(([c for c in col if c != gap] or ["+"])[0], "cv") for col in revert(alms)]
+
+
+def apply_trim(alms, idxs):
+    """
+    Basic trimming function, based on a selection of indices that are trimmed.
+    """
+    return [[row[i] for i in range(len(row)) if i not in idxs] for row in alms]
+
+
+def subsequence_of(source, target):
+    """
+    Check if source is a subsequence of target.
```

Comment:
I'd add an explanation of "subsequence" here, i.e. "all items of source appear in target in order, but possibly not consecutively". Maybe adding a doctest:
```python
>>> assert subsequence_of('abc', 'xaybzc')
```

Reply:
That's a useful definition, better than the ones I found in the literature so far.